### ISSUE TITLE
Viewer Draw fails to render correctly when using the Metal backed in macOS

### ISSUE THREAD

**Issue by corbinstreehouse** [nortikin/sverchok#5061]
## Steps to reproduce

1. Open attached file on macOS (12.5 Monterey) in Blender 3.6, using the default Metal backend instead of OpenGL

[Sverchok Erindale tut.blend.zip](https://github.com/nortikin/sverchok/files/13379683/Sverchok.Erindale.tut.blend.zip)

Actual result:

 Viewer nodes generate with some transparency, and you get a warning about using OpenGL:
[
<img width="1624" alt="Screen Shot 2023-11-16 at 7 42 13 AM" src="https://github.com/nortikin/sverchok/assets/2755434/276f1501-74f9-474b-949c-fc266139da7a">
](url)

 Expected result, when using the OpenGL backend:
<img width="1624" alt="Screen Shot 2023-11-16 at 7 41 37 AM" src="https://github.com/nortikin/sverchok/assets/2755434/c0bcded6-6d99-4285-9baa-9178a4530bff">

## Expected result

What did you expect to see as a result of those steps?

## Actual result

What did you actually get?

## Sverchok version
1.2.0

**Comment by satabol** [comment]
Sverchok is not compatible with Metal for a while. Can you select and use openGL backend?

![image](https://github.com/nortikin/sverchok/assets/14288520/4cc76376-e449-42fb-9858-aa8261b4e7ff)

**Comment by corbinstreehouse** [comment]
Yes - I am doing that. I haven’t seen any open bug reports about it not working with Metal. Can we use this one to track it? I did search for them before logging a bug.

**Comment by corbinstreehouse** [comment]
Looks like Blender 4.0+ doesn't allow one to use the OpenGL back end anymore.

**Comment by satabol** [comment]
PR #4983 for Blender 4.x is in process.
